### PR TITLE
Update YAMLS

### PR DESCRIPTION
**Pull request by abhi-mosaic**
- Turns off `progress_bar` by default to improve UX when using MCLOUD
- Update `llm/mcloud` examples to new format

### CODE REVIEW

File: llm/mcloud/mcli-1b-custom.yaml [hunk 1]
```diff
@@ -26,19 +26,20 @@ cluster: r0z0 # replace with your cluster here!
 # and referenced in our `command` above
 # We use this to customize hparams like `max_seq_len=8192` at runtime
 parameters:
-  data_remote: &data_remote ./my-copy-c4
   data_local: &data_local ./my-copy-c4
-  max_seq_len: &max_seq_len 8192
```
Comment (bmosaicml) [inline llm/mcloud/mcli-1b-custom.yaml:31]: Can you explain these changes in the PR?

**Review by bmosaicml — APPROVED** [review]

File: llm/mcloud/mcli-1b-custom.yaml [hunk 2]
```diff
@@ -26,19 +26,20 @@ cluster: r0z0 # replace with your cluster here!
...
   model:
     name: mosaic_gpt
     device: meta
     tokenizer_name: *tokenizer_name
-    d_model: 2048
-    n_heads: 16 # Modified 24->16 so that d_head == 128 to statisfy FlashAttention
-    n_layers: 24
+    d_model: 768
+    n_heads: 12
+    n_layers: 12
```
Comment (vchiley) [inline llm/mcloud/mcli-1b-custom.yaml:42]: This makes it a 125M model, not a 1B model.

File: llm/mcloud/mcli-1b-custom.yaml (hunk 1 above)
Comment (abhi-mosaic) [reply llm/mcloud/mcli-1b-custom.yaml:31]: Ah, I think as I was trying to update it, I copied over the 125M model and did not preserve some of the custom flags. Fixing it now!

**Comment by abhi-mosaic** [comment]
Reverted the `mcloud/` folder as it should not actually be changed until we have a new stable pin, and then those examples should point at the stable pin.

**Pull request merged**